bin/versions.py

```diff
@@ -18,44 +18,40 @@
 __email__ = 'slavomir.mazur@pantheon.tech'
 __version__ = '1.1.0'
 
+from configparser import ConfigParser
 from subprocess import CalledProcessError, check_output
 
 from create_config import create_config
 from pyang import __version__ as pyang_version
 from xym import __version__ as xym_version
 
 
-class ValidatorsVersions:
-    def __init__(self):
-        config = create_config()
-        confdc_exec = config.get('Tool-Section', 'confdc-exec')
-
-        # ConfD version
-        try:
-            confd_version = check_output(f'{confdc_exec} --version', shell=True).decode('utf-8').rstrip()
-        except CalledProcessError:
-            confd_version = 'undefined'
-        # yangdump version
-        try:
-            yangdump_cmd = '/usr/bin/yangdump-pro'
-            yangdump_version = check_output(f'{yangdump_cmd} --version', shell=True).decode('utf-8').strip()
-        except CalledProcessError:
-            yangdump_version = 'undefined'
-        # yanglint version
-        try:
-            yanglint_cmd = '/usr/local/bin/yanglint'
-            yanglint_version = check_output(f'{yanglint_cmd} --version', shell=True).decode('utf-8').rstrip()
-        except CalledProcessError:
-            yanglint_version = 'undefined'
-
-        self.versions = {
-            'validator_version': __version__,
-            'pyang_version': pyang_version,
-            'xym_version': xym_version,
-            'confd_version': confd_version,
-            'yanglint_version': yanglint_version,
-            'yangdump_version': yangdump_version,
-        }
-
-    def get_versions(self):
-        return self.versions
+def get_validator_versions(config: ConfigParser = create_config()) -> dict:
```

Comment:
Would it be possible to just run this code inside this module and assign the result to a constant we can import from other modules?

Reply:
That would be great, I was considering the use of some type of hashing but stopped when realized that there is a possible situation when one of the validators is updated inside the container so we need to call this function every time. If such a case is not possible I'll use a constant here

Reply:
You don't need to worry about validators being updated while the container is running, I'm pretty sure we only update those with new releases so the version shouldn't change after the container has started.